cmake/projects/SDL2/hunter.cmake

```diff
@@ -6,6 +6,17 @@ include(hunter_download)
 include(hunter_pick_scheme)
 include(hunter_cacheable)
 
+hunter_add_version(
+        PACKAGE_NAME
+        SDL2
+        VERSION
+        "2.0.4-p6"
```

Comment:
Add this version to `cmake/configs/default.cmake`:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#set-default-version

Reply:
Done